src/teleop/download_map/download_map.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import zipfile
+import json
+import boto3
+import subprocess
+
+
+# If System has issues with number of file watcher when using GUI with downloaded map
+# Run "sudo sysctl -w fs.inotify.max_user_watches=100000"
+def main():
+    result = subprocess.Popen(
+        ["rospack", "find", "mrover"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )  # runs rospack find mrover to get directory
+    m_bytes, err = result.communicate()
+    m = m_bytes.decode("utf-8")  # convert byte to string type
+    m = m[: len(m) - 1]  # the old m includes the '\n' and this gets rid of that
+    mrover = Path(m)  # load directory into path
+    path_exists = Path.exists(mrover / "src" / "teleop" / "download_map" / "keys.json")
+    if not path_exists:
+        out = {"accessKey": "", "secretKey": ""}
+        f = open(mrover / "src" / "teleop" / "download_map" / "keys.json", "w")
+        json.dump(out, f, indent=4)
+        print("Access Keys not found, input keys into keys.json")
+        exit(0)
+    f = open(mrover / "src" / "teleop" / "download_map" / "keys.json", "r")
+    keys = json.load(f)
+    s3 = boto3.client("s3", aws_access_key_id=keys["accessKey"], aws_secret_access_key=keys["secretKey"])
```

Comment:
Have you considered adding error handling to this statement or down in line 34 where `download_file()` is called? Maybe it is best to just let it error naturally but something to think about